README.md

```diff
@@ -27,77 +27,78 @@ host. Also the homepage at port 8080 contains a link to the Consul UI
 Also you can use Consul's DNS interface with e.g. dig:
 
 ```
-dig @192.168.99.100 -p 8600  order.service.consul. 
+dig @192.168.99.100 -p 8600  order.service.consul.
 dig @192.168.99.100 -p 8600  order.service.consul. ANY
 dig @192.168.99.100 -p 8600  order.service.consul. SRV
 ```
 
-Note that the demo uses the original Consul Docker template provided
+Note that the demo uses the original []Consul Docker template](https://hub.docker.com/_/consul/) provided
```

Comment:
The _ used in the URI leads to italic text afterwards. Because of that the _ must be escaped.